src/components/content/catalog/services/cancel/CancelRequestResult.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { UseMutationResult, useQueryClient } from '@tanstack/react-query';
+import { Alert } from 'antd';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { category } from '../../../../../xpanse-api/generated';
+import { catalogPageRoute } from '../../../../utils/constants';
+import { isHandleKnownErrorResponse } from '../../../common/error/isHandleKnownErrorResponse.ts';
+import { getQueryKey } from '../query/useAvailableServiceTemplatesQuery';
+
+export function CancelRequestResult({
+    category,
+    cancelServiceTemplateRequest,
+}: {
+    category: category;
+    cancelServiceTemplateRequest: UseMutationResult<void, Error, string>;
+}): React.JSX.Element | undefined {
+    const queryClient = useQueryClient();
+    const navigate = useNavigate();
+
+    const onRemove = () => {
```

Comment:
ensure that this comes back to the same service template on which the cancelled was performed.